### PR TITLE
Add support for throwable projectiles items

### PR DESCRIPTION
**Pull request by Mr-Darth**
### Description
This PR adds support for getting an manipulating the item of a throwable projectile (snowball, enderpearl etc.)

---
**Target Minecraft Versions:** 1.15.2+
**Requirements:** 1.15.2+
**Related Issues:** none

### CODE REVIEW

File: src/main/java/ch/njol/skript/util/slot/ThrowableProjectileSlot.java [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+	public ItemStack getItem() {
+		return projectile.getItem();
+	}
+	
+	@Override
+	public void setItem(@Nullable ItemStack item) {
+		projectile.setItem(item != null ? item : new ItemStack(Material.AIR));
+	}
+	
+	@Override
```
Comment (Pikachu920) [inline src/main/java/ch/njol/skript/util/slot/ThrowableProjectileSlot.java:50]: correct me if i'm wrong, but couldn't you do `projectile.getItem().getAmount()` and `project.getItem().setAmount(amount)`

**Review by Pikachu920 — CHANGES_REQUESTED** [review]

File: src/main/java/ch/njol/skript/util/slot/ThrowableProjectileSlot.java (hunk 1 above)
Comment (Mr-Darth) [reply src/main/java/ch/njol/skript/util/slot/ThrowableProjectileSlot.java:50]: I could, but the amount is always 1, so I decided to be consistent with the [ItemFrameSlot](https://github.com/SkriptLang/Skript/blob/master/src/main/java/ch/njol/skript/util/slot/ItemFrameSlot.java#L51) class.

Comment (Pikachu920) [reply src/main/java/ch/njol/skript/util/slot/ThrowableProjectileSlot.java:50]: Looks like @ShaneBeee [wrote that line](https://github.com/SkriptLang/Skript/blame/d246107d182ef1566016238c6c8d4bd199c6af00/src/main/java/ch/njol/skript/util/slot/ItemFrameSlot.java#L56) and @FranKusmiruk actually brought up the same concern that I did but was resolved. In my opinion, Skript should allow the same access to implemented APIs as Java would (with the exception of stopping you from doing things that break the server). what do you guys think @Mr-Darth @FranKusmiruk @ShaneBeee?

Comment (ShaneBeee) [reply src/main/java/ch/njol/skript/util/slot/ThrowableProjectileSlot.java:50]: While I normally agree that Skript should be more consistent with Bukkit, Skript is often doing things quite different than Bukkit would.
Also, if Im not mistaken, this method is generally used internally and not by the end user. I could be wrong.
Regardless, attempting to set the item stack amount to anything about 1, will just get ignored either way.

Comment (Mr-Darth) [reply src/main/java/ch/njol/skript/util/slot/ThrowableProjectileSlot.java:50]: As Shane said, the item amount can't be any number but 1, so everything will work the same, no matter how it's implemented.

**Review by ShaneBeee — APPROVED** [review]

**Review by Pikachu920 — APPROVED** [review]

**Pull request merged**